sdk/maps/maps-search/README.md

```diff
@@ -0,0 +1,174 @@
+for (const result of fuzzySearchResult.results) {
+  console.log(result);
+}
+```
+
+### Make a Reverse Address Search to translate coordinate location to street address
+
+You can translate coordinates into human readable street addresses. This process is also called reverse geocoding.
+This is often used for applications that consume GPS feeds and want to discover addresses at specific coordinate points.
+
+```javascript
+const coordinates: LatLon = [47.59118, -122.3327]; // [latitude, longitude]
+
+const { addresses } = await client.reverseSearchAddress(coordinates);
+for (const { address } of addresses) {
+  console.log(address);
+}
+```
+
+### Translate coordinate location into a human understandable cross street
+
+Translate coordinate location into a human understandable cross street by using Search Address Reverse Cross Street API. Most often, this is needed in tracking applications that receive a GPS feed from a device or asset, and wish to know where the coordinate is located.
+
+```javascript
+const coordinates: LatLon = [47.59118, -122.3327];
+
+const { addresses } = await client.reverseSearchCrossStreetAddress(coordinates);
+for (const { address } of addresses) {
+  console.log(address);
+}
+```
+
+## Troubleshooting
+
+### Logging
+
+Enabling logging may help uncover useful information about failures. In order to see a log of HTTP requests and responses, set the `AZURE_LOG_LEVEL` environment variable to `info`. Alternatively, logging can be enabled at runtime by calling `setLogLevel` in the `@azure/logger`:
+
+```javascript
+const { setLogLevel } = require("@azure/logger");
+setLogLevel("info");
+```
+
+For more detailed instructions on how to enable logs, you can look at the [@azure/logger package docs](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/core/logger).
+
+## Next steps
+
+Please take a look at the [samples](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/maps/maps-search/samples) directory for detailed examples on how to use this library.
+
+## Contributing
+
+If you'd like to contribute to this library, please read the [contributing guide](https://github.com/Azure/azure-sdk-for-js/tree/main/CONTRIBUTING.md) to learn more about how to build and test the code.
+
+## Related projects
+
+- [Microsoft Azure SDK for JavaScript](https://github.com/Azure/azure-sdk-for-js)
+
+![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-js%2Fsdk%2Fmaps%2Fmaps-search%2FREADME.png)
+
+[azure_cli]: https://docs.microsoft.com/cli/azure
+[azure_sub]: https://azure.microsoft.com/free/
+[azure_portal]: https://portal.azure.com
+[azure_powershell]: https://docs.microsoft.com/powershell/module/az.maps/new-azmapsaccount
+[azure_identity]: https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/identity/identity
+[defaultazurecredential]: https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/identity/identity#defaultazurecredential
```

Comment:
Need to add in the end: `![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-js%2Fsdk%2Fmaps%2Fmaps-search%2FREADME.png)`

Reply:
I think it already existed in the line 167?

Reply:
But I think we need to add one for `maps-common`

Reply:
Fixed in 19da7ce5ba213b1ce82513972b0bcca8908d34be

Reply:
Sorry my mistake. Resolved.